vagrant/provisioning/roles/alfresco/tasks/main.yml

```diff
@@ -374,6 +374,60 @@
       opencmis.server.override=true
       opencmis.server.value=https://{{ external_host }}/alfresco/api
 
+- name: change alfresco-global.properties to support second LDAP configuration
+  block:
+    - name: add ldap2 in AD/Samba section
+      become: yes
+      become_user: alfresco
+      replace:
+        path: "{{ root_folder }}/app/alfresco/shared/classes/alfresco-global.properties"
+        regexp: authentication\.chain=external1:external,ldap1:ldap-ad,alfrescoNtlm1:alfrescoNtlm
+        replace: authentication.chain=external1:external,ldap1:ldap-ad,ldap2:ldap-ad,alfrescoNtlm1:alfrescoNtlm
+    - name: remove LDAP host-specific settings section
+      become: yes
+      become_user: alfresco
+      replace:
+        path: "{{ root_folder }}/app/alfresco/shared/classes/alfresco-global.properties"
+        after: '#### LDAP host-specific settings'
+        before: '# remove unneeded Alfresco activity'
+        regexp: '.*'
+        replace: ''
+    - name: create ldap-ad folder structure
+      become: yes
+      become_user: alfresco
+      file:
+        path: "{{ item }}"
+        state: directory
+        recurse: yes
+      loop:
+        - "{{ root_folder }}/app/alfresco/shared/classes/alfresco/extension/subsystems/Authentication/ldap-ad/ldap1"
+        - "{{ root_folder }}/app/alfresco/shared/classes/alfresco/extension/subsystems/Authentication/ldap-ad/ldap2"
+    - name: create ldap1-ad.properties file and populate it 
```

Comment:
I still think we should always create the ldap1 folder with these properties, then only create ldap2 with its properties if we are deploying the foia portal.

Reply:
Also I think you can add a line to set the sync schedule, let's sync at least every hour.

Reply:
I agree with creating ldap1 folder always.
The sync should be set on 5 minutes. Otherwise, the user cannot do anything until it is synced to Alfresco. We didn't find a way to force Alfresco sync when a new user logs in to Arkcase (for Alfresco CE version).

Reply:
@kikomanaskov333  please add following to alfresco-global.properties:

nchronization.syncWhenMissingPeopleLogIn=true
synchronization.import.cron=0 0/5 * * * ? *